src/jit/morph.cpp

```diff
@@ -3993,13 +4053,12 @@ GenTreeCall* Compiler::fgMorphArgs(GenTreeCall* callNode)
 // args:
 //   call: The call whose arguments need to be morphed.
 //   hasStructArgument: Whether this call has struct arguments.
-//   
+//
 void Compiler::fgMorphSystemVStructArgs(GenTreeCall* call, bool hasStructArgument)
 {
-    unsigned flagsSummary = 0;
-    GenTreePtr      args;
-    GenTreePtr      argx;
-
+    unsigned     flagsSummary = 0;
```

Comment:
I will revert all of the whitespace changes to fgMorphSystemVStructArgs as I didn't make any changes to this method.